### ISSUE TITLE
Question: Reason for absence of BatchNorm in refine part

### ISSUE THREAD

**Issue by vfdev-5** [DrSleep/light-weight-refinenet#51]
Hi @DrSleep 

I was wondering whether there is a particular reason why there is no BatchNorm ops in your refining architecture part. There is no BatchNorm neither in original RefineNet architecture, but it can be a historical reason... Maybe you've tested your architecture with batch norms in MRF and CRP blocks and have a conclusion on that ?

Thanks

**Comment by DrSleep** [comment]
Hi, IIRC the original RefineNet was using the batch size of 1 and that is why there were no BN layers there.
I followed the same setup, although I did try including BN inside the blocks, but did not see any improvements. My batch size was still relatively small, so maybe a larger batch size coupled with synchronised BN would help.

**Comment by vfdev-5** [comment]
@DrSleep thank you for the answer !
